content/en/news/releases/1.10.x/announcing-1.10/upgrade-notes/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.10.0.
+weight: 20
+---
+
+When you upgrade from Istio 1.9 to Istio 1.10, you need to consider the changes on this page.
+These notes detail the changes which purposefully break backwards compatibility with Istio 1.9.
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio 1.9.
+
+## Inbound Forwarding Configuration
+
+The behavior of inbound forwarding will be modified in the near future. While this change is not enabled
+by default in Istio 1.10, it can be enabled today by configuring the `PILOT_ENABLE_INBOUND_PASSTHROUGH=true` environment
+variable in Istiod.
```

Comment:
I think this is not correct?  The blog said it is enabled: https://preliminary.istio.io/latest/blog/2021/upcoming-networking-changes/#future-behavior

Reply:
Changed, thanks, think the release note had old info